src/ui/actions/event-listeners.ts

```diff
@@ -277,6 +279,8 @@ export const IGNORABLE_PARTIAL_SOURCE_URLS = [
   "webpack:///src/sandbox/",
   "webpack:///sandpack-core/",
   "webpack:////home/circleci/codesandbox-client",
+  // or Cypress
+  "__cypress/runner/",
```

Comment:
Kind of new to this code– but the `"react-dom"` string above seems maybe a bit too loose without "/" around it. (Seems like it would also impact any packages/paths with "react-dom" in the name– not just the `react-dom` package itself?)

Reply:
I'm okay with that. For now the goal is primarily to jump to user code.